sdk/eventhub/azure-eventhub/TROUBLESHOOTING.md

```diff
@@ -82,11 +89,15 @@ An `AuthenticationError` means that the provided credentials do not allow for th
 
 This error can occur when an intercepting proxy is used.  We recommend testing in your hosting environment with the proxy disabled to verify.
 
-### Socket exhaustion errors
+### Managing Clients
 
-Applications should prefer treating the Event Hubs clients as a singleton, creating and using a single instance through the lifetime of their application.  This is important as each client type manages its connection; creating a new Event Hub client results in a new AMQP connection, which uses a socket.  Additionally, it is essential to be aware that your client is responsible for calling `close()` when it is finished using a client or to use the `with statement` for clients so that they are automatically closed after the flow execution leaves that block.
+Applications should manage how they open and close the eventhub client.  It is essential to be aware that your application is responsible for calling `close()` when it is finished using a client or to use the `with statement` for clients so that they are automatically closed after the flow execution leaves that block. This make sures that the socket connection to the server is closed correctly.
```

Comment:
```suggestion
Applications should manage how they open and close the eventhub client.  It is essential to be aware that your application is responsible for calling `close()` when it is finished using a client or to use the `with` statement for clients so that they are automatically closed after the flow execution leaves that block. This make sures that the socket connection to the server is closed correctly.
```

Reply:
may be helpful to add at the end: `If the client is not closed, you may see errors such as..." or other unwanted behavior?